Libraries/libcar/Headers/car/Rendition.h

```diff
@@ -44,13 +44,22 @@ class Rendition {
              * Raw data.
              */
             Data,
+            /*
+             * Extensions format
+             */
+            EXTENSIONS_WEBP,
```

Comment:
Why is this called `EXTENSIONS_WEBP` and not just `WEBP`?

Reply:
To make it clear it's non-standard